docs/xmlprofiles.rst

```diff
@@ -1274,6 +1285,39 @@ Throughput Configuration allows to limit the output bandwidth.
 |                       | bytes are allowed.                                        |            |            |
 +-----------------------+-----------------------------------------------------------+------------+------------+
 
+.. _CommonAlloc:
+
+Allocation Configuration
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Allocation Configuration allows to control the allocation behavior of internal collections for which the number
+of elements depends on the number of entities in the system.
+
+For instance, there are collections inside a publisher which depend on the number of subscribers matching with it.
+
+.. TODO: Add reference to allocation tuning section
```

Comment:
Should we address this "TODO" comment as well as part of this PR?

Reply:
The plan is to do it as part of the release PR.